sklearn/ensemble/tests/test_voting.py

```diff
@@ -560,3 +560,21 @@ def test_deprecate_none_transformer(Voter, BaseEstimator):
            "Use the string 'drop' instead.")
     with pytest.warns(DeprecationWarning, match=msg):
         est.fit(X, y)
+
+
+# TODO: Remove drop parametrize in 0.24 when None is removed in Voting*
+@pytest.mark.parametrize(
+    "Voter, BaseEstimator",
+    [(VotingClassifier, DecisionTreeClassifier),
+     (VotingRegressor, DecisionTreeRegressor)]
+)
+@pytest.mark.parametrize("drop", [None, 'drop'])
+def test_correct_named_estimator_with_drop(Voter, BaseEstimator, drop):
```

Comment:
I prefer to keep bug fix PRs limited in scope to make it easier for a reviewer.

I think a follow up PR to place this in `test_common.py` would be nice to have.

Reply:
I would agree with newcomer where it might be easier to have focus PR.

But here, it will just take twice more review time (you need to find reviewers for this PR and the next one) for adding a parametrize and move the test and we could even forget about opening the next PR.

Reply:
Okay, I'll move it

Reply:
Hmm there is no `test_common` in `ensemble`. Creating a `test_common` just for `Voting` and `Stacking` in `ensemble` seems slightly strange.

Reply:
Uhm apparently I did not commit my `test_common.py` when refactoring. So a second review will be required in this case :)